### ISSUE TITLE
Time to update

### ISSUE THREAD

**Issue by thednp** [fleck/bootstrap.native-loader#7]
I've updated the build scripts for BSN, basically I've unified build scripts to use a single file for both V3 and V4, so please check my updated [WIKI](https://github.com/thednp/bootstrap.native/wiki/How-to-use#custom-builds) if you have the time. 

Basically

```javascript
const bsn3 = require('bootstrap.native/build-module.js')
let bsn = require('bootstrap.native/build-module-v4.js')
```
becomes
```javascript
const bsn = require('bootstrap.native/lib/build-module.js')
```

Also 
```javascript
  if (options.bsVersion === 3) {
    bsn = bsn3
  }
```
can be removed.

Now a usecase to be showcased:

```javascript

{
  test: /bootstrap\.native/,
  use: {
    loader: 'bootstrap.native-loader',
    options: {
      only: ['modal', 'dropdown'],
      bs_version: '4' // default is '3'
    }
  }
}
```

Please update, test and let me know of any issue.

Best of luck

**Comment by fleck** [comment]
Just published 2.0.0 to NPM!

**Comment by thednp** [comment]
@fleck I'm not going to open a new issue, I will keep you posted here: please check bootstrap.native latest master, and prepare for a new version.

Happy New Year!
